src/utils/gen-chain-yaml.ts

```diff
@@ -0,0 +1,47 @@
+import { writeFile, unlink } from "fs/promises"
+import { join } from "path"
+import config from '../../../grants-contracts-upgradeable/config.json'
+import { ChainData } from "../types"
+import { readYaml, writeYaml } from "./yaml"
+import { chainsDirectory } from '../config.json'
+import getArg from "./get-arg"
+
+
+const TEMP_CONFIG_PATH = '././contracts/evm'
+
+const generateChainYaml = async (chainName: string | undefined) => {
+
+    // Read yaml file from inside chains folder
+    const yamlFilePath = join(chainsDirectory, `${chainName}.yaml`)
+    let yaml = await readYaml<ChainData>(yamlFilePath)
+
+    // Write the values of contract addresses from config to the yaml file
+    const qbContracts = {
+        "applications": {
+            "address": config.applicationRegistryAddress.proxy,
+            "startBlock": 1,
```

Comment:
We def need the right start block otherwise the subgraph will take forever to sync

Reply:
Yes, will update the script in the contracts repo to get the startBlock numbers.